### PR TITLE
Avoid `flutter pub get` failing on example directory.

### PR DESCRIPTION
**Pull request by isoos**

### CODE REVIEW

File: lib/src/utils.dart [hunk 1]
```diff
@@ -244,4 +244,23 @@ extension ProcessResultExt on ProcessResult {
...
+  /// (both converted to [String]), with limits on individual line
+  /// lengths and total lines. Total length should not be more than 4KiB.
+  String get asTrimmedOutput {
+    Iterable<String> firstFewLines(String type, String output) sync* {
+      if (output.isEmpty) return;
+      yield '$type:';
+      yield* const LineSplitter()
+          .convert(output)
+          .take(10)
+          .map((e) => e.length > 200 ? e.substring(0, 200) : e);
```
Comment (sigurdm) [inline lib/src/utils.dart:258]: Could we make the logic here, such that it is clear that the output was truncated (eg. [...200 more lines])

**Review by sigurdm — APPROVED** [review]
Nice
LGTM

**Comment by sigurdm** [comment]
There are a few new test failures - we should probably look into them

**Review by jonasfj — APPROVED** [review]

**Comment by isoos** [comment]
> There are a few new test failures - we should probably look into them

I'm on it. I'll leave the license test failures, but fix the end2end and version test.

**Comment by isoos** [comment]
Not the only failing test is the known issue with the license detection. Merging...

**Pull request merged**